workspace/read-manifest/src/catalogs.ts

```diff
@@ -57,9 +57,3 @@ export function assertValidWorkspaceManifestCatalogs (manifest: { packages?: rea
     }
   }
 }
-
-export function checkDefaultCatalogIsDefinedOnce (manifest: { catalog?: WorkspaceCatalog, catalogs?: WorkspaceNamedCatalogs }): void {
```

Comment:
Moving this check out of `readWorkspaceManifest()` and into `getCatalogsFromWorkspaceManifest()`. I think it's generally a good idea to perform parsing and validation in the same step.